actors/builtin/miner/policy.go

```diff
@@ -117,6 +117,10 @@ const MinSectorExpiration = 180 * builtin.EpochsInDay
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
 const MaxSectorExpirationExtension = 540 * builtin.EpochsInDay
 
+// MaxPoStChainCommitAge is the maximum distance back that a PoSt can commit to
+// a chain before being invalid
+const MaxPoStChainCommitAge = 5
```

Comment:
@nicola has indicated this isn't the right number, so we won't merge until we have a better one.